tests/tensorflow/helpers.py

```diff
@@ -84,10 +86,25 @@ def get_basic_n_conv_test_model(input_shape=(24, 24, 1), in_out_ch=((1, 3), (3,
     return tf.keras.Model(inputs=inputs, outputs=outputs)
 
 
-def create_compressed_model_and_algo_for_test(model, config, should_init=True):
+class DummyCompressionState(CompressionState):
+    @property
+    def builder_state(self) -> Dict:
+        return {}
+
+    @property
+    def ctrl_state(self) -> Dict:
+        return {}
+
+    def load_state(self, state: Dict):
+        pass
+
+
+def create_compressed_model_and_algo_for_test(model, config, compression_state=None, force_no_init=False):
     assert isinstance(config, NNCFConfig)
     tf.keras.backend.clear_session()
-    algo, model = create_compressed_model(model, config, should_init)
+    if force_no_init:
+        compression_state = DummyCompressionState()
+    algo, model = create_compressed_model(model, config, compression_state)
```

Comment:
This is a trick just for the test purposes to create compressed model without checkpoint and without initialization.

User can do it by explicitly disabling initialization in the config: `range_init_samples=0`